src/cromshell/utilities/http_utils.py

```diff
@@ -35,7 +35,9 @@ def assert_can_communicate_with_server(config):
 
 
 def check_http_request_status_code(
-    short_error_message: str, response: requests.models.Response
+    short_error_message: str,
+    response: requests.models.Response,
+    raise_exception: bool = True,
```

Comment:
Added this parameter so that the function can be called without throwing an error. By default, this parameter is set to `True` which means the function by default throws an error.